geocoding/lib/geocoding.dart

```diff
@@ -36,13 +35,28 @@ Future<List<Location>> locationFromAddress(
 /// When not supplied the currently active locale of the device will be used.
 /// The `localeIdentifier` should be formatted using the syntax:
 /// [languageCode]_[countryCode] (eg. en_US or nl_NL).
-Future<List<Placemark>> placemarkFromCoordinates(
+Future<List<Placemark>>? placemarkFromCoordinates(
   double latitude,
   double longitude, {
   String? localeIdentifier,
 }) =>
-    GeocodingPlatform.instance.placemarkFromCoordinates(
+    GeocodingPlatform.instance?.placemarkFromCoordinates(
       latitude,
       longitude,
-      localeIdentifier: localeIdentifier,
     );
+
+/// Returns a list of [Location] instances found for the supplied address.
+///
+/// In most situations the returned list should only contain one entry.
+/// However in some situations where the supplied address could not be
+/// resolved into a single [Location], multiple [Location] instances may be
+/// returned.
+///
+/// Optionally you can specify a locale in which the results are returned.
+/// When not supplied the currently active locale of the device will be used.
+/// The `localeIdentifier` should be formatted using the syntax:
+/// [languageCode]_[countryCode] (eg. en_US or nl_NL).
+Future<bool>? isPresent({
```

Comment:
This should return a "nullable" `Future`.
```suggestion
Future<bool> isPresent({
```